hardhat.config.ts

```diff
@@ -37,8 +37,8 @@ const config: HardhatUserConfig = {
       zksync: true,
     },
     inMemoryNode: {
-      url: "http://localhost:8011",
-      ethNetwork: "", // in-memory node doesn't support eth node; removing this line will cause an error
+      url: "http://127.0.0.1:8011",
+      ethNetwork: undefined, // in-memory node doesn't support eth node; removing this line will cause an error
```

Comment:
Why is this necessary?

Reply:
The JS lsp suggested to use `undefined`, but i'd rather use an empty string here.